sos/report/plugins/ipmitool.py

```diff
@@ -27,12 +27,17 @@ def setup(self):
             cmd += " -I usb"
 
         self.add_cmd_output([
+            "%s channel info 3" % cmd,
+            "%s channel getaccess 3" % cmd,
```

Comment:
What's the output of this command? I was under the impression that `getaccess` had to do with authorizing access, which is not something we'd want to automatically do when a user runs `sos`.

Reply:
```
$ cat ipmitool_channel_info_3
Channel 0x3 info:
  Channel Medium Type   : 802.3 LAN
  Channel Protocol Type : IPMB-1.0
  Session Support       : multi-session
  Active Session Count  : 13
  Protocol Vendor ID    : 7154
  Volatile(active) Settings
    Alerting            : enabled
    Per-message Auth    : enabled
    User Level Auth     : enabled
    Access Mode         : always available
  Non-Volatile Settings
    Alerting            : enabled
    Per-message Auth    : enabled
    User Level Auth     : enabled
    Access Mode         : always available
$ cat ipmitool_channel_getaccess_3
Maximum User IDs     : 15
Enabled User IDs     : 3

User ID              : 1
User Name            :
Fixed Name           : Yes
Access Available     : call-in / callback
Link Authentication  : disabled
IPMI Messaging       : disabled
Privilege Level      : NO ACCESS
Enable Status        : disabled

User ID              : 2
User Name            : root
Fixed Name           : No
Access Available     : call-in / callback
Link Authentication  : enabled
IPMI Messaging       : enabled
Privilege Level      : ADMINISTRATOR
Enable Status        : disabled
...
```

It does not store password.